mlir/CMakeLists.txt

```diff
@@ -34,3 +34,46 @@ add_subdirectory(lib)
 add_subdirectory(lib/CAPI)
 add_subdirectory(test)
 add_subdirectory(tools)
+
+if (NOT LLVM_INSTALL_TOOLCHAIN_ONLY)
+  install(DIRECTORY include/mlir include/mlir-c
```

Comment:
Does all those files below needed when build with `-DBUILD_FAT_LIBMLIRMIOPEN`?

Reply:
Yes but we didn't install the header files which is also needed by the external users.